spyder/plugins/ipythonconsole/comms/tests/test_console_kernel.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Mon Aug  5 16:43:33 2019
+
+@author: quentinpeter
+"""
+from ipykernel.tests.test_embed_kernel import setup_kernel
+from qtconsole.comms import CommManager
+import pytest
+
+from spyder.plugins.ipythonconsole.comms.kernelcomm import KernelComm
+from spyder_kernels.py3compat import PY3, to_text_string
+
+
+TIMEOUT = 15
+
+
+def test_runcell(tmpdir):
```

Comment:
Is it possible for you add tests for `debugcell` here? I don't want to add such an important new functionality without being tested.

Reply:
I'll do that soon